ticketbuyer/tb.go

```diff
@@ -313,16 +313,6 @@ func (tb *TB) buy(ctx context.Context, passphrase []byte, tip *wire.BlockHeader,
 	return err
 }
 
-// AccessConfig runs f with the current config passed as a parameter.  The
-// config is protected by a mutex and this function is safe for concurrent
-// access to read or modify the config.  It is unsafe to leak a pointer to the
-// config, but a copy of *cfg is legal.
-func (tb *TB) AccessConfig(f func(cfg *Config)) {
```

Comment:
this change is fine if we we leave this method, I don't see a reason to remove it.